server/index.js

```diff
@@ -1,46 +1,41 @@
 const express = require('express');
-const { db } = require('../database/index.js');
+const morgan = require('morgan');
+const Promise = require('bluebird');
+const mongoose = require('mongoose');
+mongoose.Promise = require('bluebird');
 const { destinationModel } = require('../database/destinationDB.js');
 const { experienceModel } = require('../database/experiencesDB.js');
-const _ = require('lodash');
+
 // const axios = require('axios');
-const morgan = require('morgan');
 // const { client } = require('../s3client.js');
-const mongoose = require('mongoose');
-
-mongoose.Promise = require('bluebird');
-const Promise = require('bluebird');
 
 const app = express();
 const port = 3636;
 
 app.use(morgan('dev'));
 app.use(express.static('/Users/dustinmorris/junior/trip/public/dist'));
 
-const findExperiences = function(destination) {
-  return Promise.map(destination[0].experiences, (id) => {
-    return experienceModel.find({id: `${id}`})
-    .then((experience) => experience[0])
-    .catch((error) => {
-      return {};
-    })
-  });
-};
+const findExperiences = (destination) => (
+  Promise.map(destination[0].experiences, (id) => (
+    experienceModel.find({ id: `${id}` })
+      .then((experience) => experience[0])
+      .catch(() => ({}))
```

Comment:
If an error is passed to the catch block it may be valuable to log the error or track the location.